.github/workflows/createNewVersion.yml

```diff
@@ -45,6 +45,11 @@ jobs:
     outputs:
       NEW_VERSION: ${{ steps.bumpVersion.outputs.NEW_VERSION }}
     steps:
+      - name: Validate actor
```

Comment:
Might as well validate the actor before the checkout or turnstyle. The less time between the checkout and the push, the lower the likelihood of the race condition occurring.